a = 0
try:
 b = 1/a
except Exception as e:
 print(e)
 print(123)


 a = 5

 if (a > 2):

     print(234)
print("Finally concept starts")
a=0
b=5
try:
    c = b/a
    print(c)
except Exception as e:
    print(e)
finally:
    print('Finally always executed')





